db/migrate/20170913145421_create_orchestrations.rb

```diff
@@ -0,0 +1,13 @@
+class CreateOrchestrations < ActiveRecord::Migration
+  def change
+    create_table :orchestrations do |t|
```

Comment:
@ereslibre could you bump the migration timestamp? as we recently added another one, it wont run.